.github/workflows/GCC.yml

```diff
@@ -0,0 +1,91 @@
+# This is a CI workflow for the fv3atm project.
+#
+# This workflow builds and tests the fv3atm library using GCC, and it tests
+# different CMake build options.
+#
+# Alex Richert, 6 Dec 2023
+
+name: GCC
```

Comment:
What does GCC stand for?

Reply:
GNU Compiler Collection - it's the GNU compilers.

Reply:
Sorry, I thought it's something else.